srv/components/system/config/set_hostname.sls

```diff
@@ -15,9 +15,12 @@
 # please email opensource@seagate.com or cortx-questions@seagate.com.
 #
 
+Update hostname file permissions:
+  cmd.run:
+    - name: "chattr -i /etc/hostname"
+
 Set hostname:
-  network.system:
```

Comment:
after changing the attr can we still use network.system way to update hostname? Or is there any other salt standard module to do the same?

Reply:
@83bhp 
yes we can use `network.system`, verified and updated PR accordingly